decidim-proposals/app/commands/decidim/proposals/admin/create_proposal.rb

```diff
@@ -58,6 +58,9 @@ def create_proposal
             action_user: form.current_user
           )
           @attached_to = @proposal
+          Decidim.traceability.perform_action!(:publish, @proposal, form.current_user, visibility: "all") do
+            @proposal.update!(published_at: Time.current)
+          end
```

Comment:
Actually this is caused by the fact that proposal's versions are being filtered out at the Presenter level, and in the template, we called the `versions` on the raw resource, instead of the filtered one. [aea2cb0](https://github.com/decidim/decidim/pull/13289/commits/aea2cb0914dcb4532f7cbfa03fe02d68e2b3c3aa) and [f2314d6](https://github.com/decidim/decidim/pull/13289/commits/f2314d61fd5258c657cb209364c23e24096c7efe) should fix the issue .

Reply:
[bbe7934](https://github.com/decidim/decidim/pull/13289/commits/bbe7934245df4f3da1e025f8f68b43810fb8c2b7) Added the spec to validate the fix.

Reply:
Great! I can confirm the fix